concepticondata/conceptlists.tsv

```diff
@@ -275,3 +275,4 @@ Othaniel-2017-300	Othaniel, Nlabephee Kefas	2017		300	basic	English	Jen language
 Nordlinger-2019-57	Nordlinger, Rachel and Thieberger, Nick	2019		57	basic	English	Australian	https://arts.unimelb.edu.au/research-unit-for-indigenous-language/research/current-research-projects/50-words-project	Nordlinger2019		The given list underlies the [50 words project](https://arts.unimelb.edu.au/research-unit-for-indigenous-language/research/current-research-projects/50-words-project), initiated by the Research Unit for Indigenous Language and aimed at collecting data on Australian languages for, primarily, teaching purposes. Note, however, that, despite the name of the project, the list represents 57 lexical items.		
 Ritchie-2019-111	Ritchie, Sandy and Sproat, Richard and Gorman, Kyle and van Esch, Daan and Schallhart, Christian and Bampounis, Nikos and Brard, Benoît and Fromseier Mortensen, Jonas and Holt, Millie and Mahon, Eoin	2019		111	questionnaire	English	Global	https://www.isca-speech.org/archive/Interspeech_2019/pdfs/2807.pdf	Ritchie2019		This list represents 111 numerals, used in the development of computational solutions to number verbalization in various languages. This initiative is part of a broader research effort at Google, enhancing the available machine learning techniques for building number names grammars across a wide range of languages for application in automatic speech recognition and text-to-speech synthesis systems.	3530-3534	
 Joo-2019-100	Joo, Ian	2019		100	ranked, basic	English	Global		Joo2019		This list of 100 items is based on the [Leipzig-Jakarta concept list](:ref:Tadmor-2009-100) and was employed in the study of phonosemantic patterns in the basic vocabulary of 66 genealogically unrelated languages.	1-12	
+Tolmie-1884-211	Tolmie, William F. and Dawson, George M.	1884		211	basic	English	Languages of the Native Tribes of British Columbia	https://open.library.ubc.ca/collections/bcbooks/items/1.0222394	Tolmie1884	Tolmie1884	This collection of 211 items, based on [Instructions for Research Relative to the Ethnology and Philology of America](:bib:Gibbs1863), was collected by the authors during expeditions in the British Columbia territory during the winters of 1875 and 1876. It contains basic vocabulary terms in addition to local flora and fauna terms. The core list of 211 items, also known as Dawson-Tolmie Vocabularies, is extended by appendices with extended vocabularies for specific languages in the area, but is not considered part of the main vocabulary list.		
```

Comment:
If they are also know as Dawson-Tolmie Vocabs, you could add this as an alias for the list.